controls/postgres_spec.rb

```diff
@@ -23,43 +22,14 @@
 
 title 'PostgreSQL Server Configuration'
 
-# attributes
-
-USER = attribute(
-  'user',
-  description: 'define the postgresql user to access the database',
-  default: 'postgres'
-)
-
-PASSWORD = attribute(
-  'password',
-  description: 'define the postgresql password to access the database',
-  default: 'iloverandompasswordsbutthiswilldo'
-)
-
-POSTGRES_DATA = attribute(
-  'postgres_data',
-  description: 'define the postgresql data directory',
-  default: postgres.data_dir
-)
-
-POSTGRES_CONF_DIR = attribute(
-  'postgres_conf_dir',
-  description: 'define the postgresql configuration directory',
-  default: postgres.conf_dir
-)
-
-POSTGRES_CONF_PATH = attribute(
-  'postgres_conf_path',
-  description: 'define path for the postgresql configuration file',
-  default: postgres.conf_path
-).to_s
-
-POSTGRES_HBA_CONF_FILE = attribute(
-  'postgres_hba_conf_file',
-  description: 'define path for the postgresql configuration file',
-  default: File.join(postgres.conf_dir.to_s, 'pg_hba.conf')
-)
+# inputs
+USER = input('user', value: 'postgres')
```

Comment:
I think, I'm in favor of keeping the descriptions, or is there something that's keeping us?